sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -779,6 +779,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val PARTITION_BATCH_SIZE = buildConf("spark.sql.catalog.partitionBatchSize")
```

Comment:
@cloud-fan Any suggestion on config name? spark.sql.hive.partitionBatchSize/ spark.sql.catalog.partitionBatchSize ?

Reply:
Does this really need to be configurable - will it make a meaningful difference to use values other than 100?

Reply:
@srowen  @cloud-fan  I did some experiments with different batch sizes 100/200/500 and the total time to add 10K partitions was almost same in all 3 cases.

so makint it configurable doesn't make much sense. What do you suggest?

Reply:
I'd just set it to a largeish value, like 100 - something the metastore can comfortably handle in one shot.

Reply:
done...